ui/apps/platform/cypress/integration/vulnerabilities/nodeCves/cveDetailPage.test.ts

```diff
@@ -0,0 +1,73 @@
+import withAuth from '../../../helpers/basicAuth';
+import { hasFeatureFlag } from '../../../helpers/features';
+
+describe('Node CVEs - CVE Detail Page', () => {
+    withAuth();
+
+    before(function () {
+        if (!hasFeatureFlag('ROX_VULN_MGMT_NODE_PLATFORM_CVES')) {
+            this.skip();
+        }
+    });
+
+    it('should restrict access to users with insufficient permissions', () => {
+        // check that users without Node access cannot access the Node CVE Detail page directly
+    });
+
+    it('should only show relevant filters for the Node CVE Detail page', () => {
+        // check the advanced filters and ensure only the relevant filters are displayed
+    });
+
+    it('should link to the correct pages', () => {
+        // clicking the Node CVEs breadcrumb should navigate to the overview page with the CVE tab selected
+        // clicking a Node name in the list should navigate to the correct Node details page
+    });
+
+    it('should sort Node table columns', () => {
+        // check sorting of Node column
+        // check sorting of CVE Severity column
+        // check sorting of CVE status column
+        // check sorting of CVSS column
+        // check sorting of cluster column
+        // check sorting of Operating System column
+        // check sorting of Affected components column
+    });
+
+    it('should filter the Node table', () => {
+        // filtering by Node name should only display rows with a matching name
+        // filtering by CVE Severity should only display rows with a matching severity
```

Comment:
As a result of your expert investigation, classic test conditionally skip some of these scenarios in OpenShift environment. Therefore, it might be necessary to split the test.

Reply:
Do you remember why it is that the VM 1.0 tests need to be skipped in OCP?

I'll add a note in this outline so that we don't forget to check this case, but it would be great if we could have the same coverage in OpenShift jobs as well.